.github/workflows/cd.yml

```
@@ -0,0 +1,23 @@
+name: cd
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  release-next:
+    name: release-next
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```
There is not need to use 20.04. In addition, 20.04 will become latest soon.